cmd/swarmctl/cluster/update.go

```diff
@@ -136,5 +143,6 @@ func init() {
 	updateCmd.Flags().StringSlice("secret", nil, "Secret required to join the cluster")
 	updateCmd.Flags().Int64("taskhistory", 0, "Number of historic task entries to retain per slot or node")
 	updateCmd.Flags().Duration("certexpiry", 24*30*3*time.Hour, "Duration node certificates will be valid for")
+	updateCmd.Flags().StringSlice("external-ca-url", nil, "URL to one or more CFSSL CA certificate signing endpoints")
```

Comment:
I would vote that it should be singular as the majority of other command options which can be plural are predominantly singular. For example with `run`: `--security-opt`, `--dns-opt`, and `--log-opt` are all singular.